src/main/js/components/contextMenu/ColumnContextMenu.jsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Context menu for column options. Opened by ColumnEntry.
+ */
+
+import React from "react";
+import GenericContextMenu from "./GenericContextMenu";
+import TableauxConstants from "../../constants/TableauxConstants";
+import listensToClickOutside from 'react-onclickoutside/decorator';
+import * as AccessControl from "../../helpers/accessManagementHelper";
+import {compose, contains} from 'lodash/fp';
+const Alignments = TableauxConstants.Alignments;
+import ActionCreator from "../../actions/ActionCreator";
+import i18n from "i18next";
+
+const PROTECTED_CELL_KINDS = ['concat']; //cell kinds that should not be editable
+
+@listensToClickOutside
+class ColumnContextMenu extends React.Component {
+  constructor(props) {
+    super(props);
+    const {x, y} = props;
+    this.state = {
+      x: x,
+      y: y
+    }
+  }
+
+  handleClickOutside = evt => {
+    const el = document.getElementById(this.props.colId);
```

Comment:
could please document what you are doing there and that you tried `ignoreOutsideClick`. also `colId` is a bit misleading